cpu/kinetis_common/include/mcg.h

```diff
@@ -130,7 +130,7 @@ int kinetis_mcg_set_mode(kinetis_mcg_mode_t mode);
 }
 #endif
 
-#endif /* __MCG_CPU_H */
+#endif /* KINETIS_CPU_USE_MCG */
 /** @} */
 
-#endif /* KINETIS_CPU_USE_MCG */
+#endif /* MCG_CPU_H */
```

Comment:
@miri64 Martine, could you help check here. I though ```/* KINETIS_CPU_USE_MCG */``` and ```/* MCG_CPU_H */``` were misplaced in this file, and I made this revision. But I am not sure about it.